_data/simple-icons.json

```diff
@@ -1070,6 +1070,11 @@
             "hex": "72EF36",
             "source": "https://www.gumtree.com"
         },
+        {
+            "title": "Hackaday",
+            "hex": "DDDDDD",
+            "source": "https://hackaday.com/"
+        },		
```

Comment:
Can you remove the trailing whitespace on this line? It's not much but we prefer not to have it there 🙂 

---

<sup>Linting for this is being implemented in #1349</sup>